start.go

```diff
@@ -89,37 +99,42 @@ func start(c *cli.Context) {
 		// Check there wasn't a problem initiating the Lambda runtime
 		if err != nil {
 			if err == ErrRuntimeNotSupported {
-				log.Printf("Ignoring %s (%s) due to unsupported runtime (%s)\n", name, function.Handler, function.Runtime)
+				warnMsg.Printf("Ignoring %s (%s) due to unsupported runtime (%s)\n", name, function.Handler, function.Runtime)
 				continue
 			} else {
-				log.Printf("Ignoring %s (%s) due to %s runtime init error: %s\n", name, function.Handler, function.Runtime, err)
+				warnMsg.Printf("Ignoring %s (%s) due to %s runtime init error: %s\n", name, function.Handler, function.Runtime, err)
 				continue
 			}
 		}
 
 		// Add this AWS::Serverless::Function to the HTTP router
 		if err := mux.AddFunction(&function, runt.InvokeHTTP(c.String("profile"))); err != nil {
 			if err == router.ErrNoEventsFound {
-				log.Printf("Ignoring %s (%s) as no API event sources are defined", name, function.Handler)
+				warnMsg.Printf("Ignoring %s (%s) as no API event sources are defined\n", name, function.Handler)
 			}
 		}
-
 	}
 
 	// Check we actually mounted some functions on our HTTP router
 	if len(mux.Mounts()) < 1 {
 		if len(functions) < 1 {
-			fmt.Fprintf(stderr, "ERROR: No Serverless functions were found in your SAM template.\n")
+			errMsg.Fprintf(stderr, "ERROR: No Serverless functions were found in your SAM template.\n")
 			os.Exit(1)
 		}
-		fmt.Fprintf(stderr, "ERROR: None of the Serverless functions in your SAM template were able to be mounted. See above for errors.\n")
+		errMsg.Fprintf(stderr, "ERROR: None of the Serverless functions in your SAM template were able to be mounted. See above for errors.\n")
 		os.Exit(1)
 	}
 
 	fmt.Fprintf(stderr, "\n")
 
 	for _, mount := range mux.Mounts() {
-		msg := fmt.Sprintf("Mounting %s (%s) at http://%s:%s%s %s", mount.Function.Handler, mount.Function.Runtime, c.String("host"), c.String("port"), mount.Path, mount.Methods())
+		if mount.Function == nil || len(mount.Function.Handler) == 0 {
```

Comment:
you mount the missingFunctionHandler right? then the length will never be zero

Reply:
Assuming it all goes well in the merge. This is a sanity check. Without this here we throw the segfault mentioned in issue #225